openrtb3/bool.go

```diff
@@ -0,0 +1,48 @@
+package openrtb3
+
+import (
+	"bytes"
+)
+
+// Bool is a bool type wrapper.
+type Bool bool
```

Comment:
TODO: tests, including embedded Bool field with "omitempty" option.

Reply:
TODO: think if this is REALLY needed (does it REALLY help much or just a child of engineer's boredom?..)

Reply:
Removed, no need to make this lib too smart.